src/modules/UI/scenes/SendConfirmation/action.js

```diff
@@ -125,43 +136,70 @@ export const updateWalletTransfer = (wallet: AbcCurrencyWallet) => (dispatch: an
   dispatch(updateLabel(wallet.name))
 }
 
-export const processParsedUri = (parsedUri: AbcParsedUri) => (dispatch: any, getState: any) => {
+export const makeSpend = (options: AbcMakeSpendInfo) => (dispatch: any, getState: any) => {
   const state = getState()
   const walletId = UI_SELECTORS.getSelectedWalletId(state)
   const abcWallet = CORE_SELECTORS.getWallet(state, walletId)
-  const spendInfo: AbcSpendInfo = makeSpendInfo(parsedUri)
-  spendInfo.networkFeeOption = state.ui.scenes.sendConfirmation.feeSetting
-  spendInfo.customNetworkFee = state.ui.scenes.sendConfirmation.feeSatoshi
-
-  return WALLET_API.makeSpend(abcWallet, spendInfo)
+  return WALLET_API.makeSpend(abcWallet, makeSpendInfo(options))
   .then((abcTransaction: AbcTransaction) => {
-    dispatch(updateTransactionAction(parsedUri, abcTransaction, null))
+    dispatch(updateTransaction(abcTransaction))
```

Comment:
Do not make 2 dispatch calls. These can be one call with two parameters. Save two separate renders happening instead of one

Reply:
Why did this even need to be changed?

Reply:
It's fixed in the new version I'm currently testing